README.md

```diff
@@ -431,4 +442,34 @@ $ operator-sdk up local
 ```
 ## Releases  
 
-The latest pravega releases can be found on the [Github Release](https://github.com/pravega/pravega-operator/releases) project page.
+The latest Pravega releases can be found on the [Github Release](https://github.com/pravega/pravega-operator/releases) project page.
+
+## Troubleshooting
+
+### Helm Error: no available release name found
+
+This section would guide the users when installing nfs-server-provisioner, if they encounter any error on failure of `helm install`.
```

Comment:
Replace this whole section with:

---

When installing a cluster for the first time using `kubeadm`, the initialization defaults to setting up RBAC controlled access, which messes with permissions needed by **Tiller** to do installations, scan for installed components, and so on. `helm init` works without issue, but `helm list`, `helm install` and other commands do not work.

```
$ helm install stable/nfs-server-provisioner
Error: no available release name found
```

The following workaround can be applied to resolve the issue:

1. Create a service account for the Tiller.

```
kubectl create serviceaccount --namespace kube-system tiller
```

2. Bind that service account to the `cluster-admin` ClusterRole.

```
kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
```

3. Add the service account to the Tiller deployment.

```
kubectl patch deploy --namespace kube-system tiller-deploy -p '{"spec":{"template":{"spec":{"serviceAccount":"tiller"}}}}'
```

The above commands should resolve the errors and `helm install` should work correctly.

Reply:
Accepted suggestion